pkg/heartbeat/heartbeat.go

```diff
@@ -162,10 +156,21 @@ func UserAgent(plugin string) string {
 		log.Debugf("goInfo.GetInfo error: %s", err)
 	}
 
+	os := runtime.GOOS
+	alternateOS := osName()
```

Comment:
If `osName()` is moved to a more meanful package then this function can embed the logic for `runtime.GOOS` and there's no need to do `if alternateOS != "" && !strings.EqualFold(alternateOS, os)` here.